### ISSUE TITLE
Explain why sign button is disabled

### ISSUE THREAD

**Issue by Sjors** [Sjors/nthkey-ios#30]
The sign button is disabled when `psbt.canSign` returns false. We should make it more clear why this happens. This may require a PR to `libwally-swift`.

`canSign` is implemented here:
https://github.com/blockchain/libwally-swift/blob/master/LibWally/PSBT.swift#L54-L73

Some reasons why it can fail:
* wrong network: the network is not specified in the PSBT, but you can tell from the derivation path, e.g. `m/84h/0h` is mainnet and `m/84h/1h` is testnet
* change address derivation is not what we expect
* we don't have matching keys (wrong derivation path?); this might be where we should check if any of the other wallets do match

**Comment by w-i-n-s** [comment]
some details https://github.com/Sjors/nthkey-ios/issues/53#issuecomment-860060117

**Comment by w-i-n-s** [comment]
@Sjors Yes, we need additional commits which adds more information to the returned value. Instead of just returning `nil` as an error-cased result. Some swift [language details](https://docs.swift.org/swift-book/LanguageGuide/ErrorHandling.html) here.

Personally, I would recommend use `Result<[PubKey: KeyOrigin], PBSTSignError>` 
where `enum PBSTSignError: Error {}`
